packages/vm/src/params.ts

```diff
@@ -54,8 +55,14 @@ export const paramsVM: ParamsDict = {
 .  * Shard Blob Transactions
 .  */
   4844: {
+    // gasConfig
+    targetBlobGasPerBlock: 393216, // The target blob gas consumed per block
     blobGasPerBlob: 131072, // The base fee for blob gas per blob
     maxBlobGasPerBlock: 786432, // The max blob gas allowable per block
+    blobGasPriceUpdateFraction: 3338477, // The denominator used in the exponential when calculating a blob gas price
+    // gasPrices
+    simplePerBlobGas: 12000, // The basic gas fee for each blob
```

Comment:
Can you remove this one? It is unused. See: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3836